src/alchemy-apis/types.ts

```diff
@@ -135,6 +137,20 @@ export interface GetNftsResponse {
   totalCount: number;
 }
 
+export interface TransactionReceiptsBlockNumber {
+  blockNumber: string;
+}
+export interface TransactionReceiptsBlockHash {
+  blockHash: string;
+}
+export type TransactionReceiptsParams =
+  | TransactionReceiptsBlockNumber
+  | TransactionReceiptsBlockHash;
+
+export interface TransactionReceiptsResponse {
+  receipts: TransactionReceipt[] | null;
+}
+
```

Comment:
+1 to @kurushdubash's point of handling error cases for the block not existing or the servers being down. FWIW the error responses look like this in [composer](https://dashboard.alchemyapi.io/composer?composer_state=%7B%22chain%22%3A0%2C%22network%22%3A0%2C%22methodName%22%3A%22alchemy_getTransactionReceipts%22%2C%22paramValues%22%3A%5B%7B%22blockNumber%22%3A%220xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF%22%7D%5D%7D): 
<img width="1375" alt="Screen Shot 2022-02-03 at 7 47 45 AM" src="https://user-images.githubusercontent.com/3498866/152377458-4bb45224-23a7-45f4-bad1-35635cd21b22.png">